### DISCUSSION TITLE
CLI - Write back metadata while exporting files

### DISCUSSION THREAD

**Discussion by ua741** [ente-io/ente#721] [Enhancements]
Currently, if you add any caption or change date time, those changes are not reflected back in the file.

Integrate with `exiftool` so that we can easily write back any such changes to the file.

**Comment by floschoepfer** [comment]
+1

**Reply by StefanOltmann** [reply]
This is very important.

**Comment by petazonk** [comment]
Also, face data should be written back to the file or at least the name of the person.
